app/scripts/constants.js

```diff
@@ -257,6 +257,23 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
                 "/create-route",
                 "/edit/routes/"
               ]
+            },
+            {
+              label: "Provisioned Services",
+              href: "/browse/service-instances",
+              prefixes: [
+                "/browse/service-instances/"
+              ],
+              isValid: function(){
+                return _.get(window.OPENSHIFT_CONSTANTS, 'ENABLE_TECH_PREVIEW_FEATURE.service_catalog_landing_page');
+              },
+              canI: {
+                resource: {
+                  group: 'servicecatalog.k8s.io',
+                  resource: 'instances'
```

Comment:
This will need to change to `'serviceinstances'` when https://github.com/openshift/origin/pull/16150 is merged.